docs/protocols/cota_main.md

```diff
@@ -14,7 +14,7 @@ The CoTA Cell simply consists of two necessary parts, the SMT root in the `cell
  # CoTA cell data structure
 data:
     version: byte
-    smt_root: byte32
+    smt_root: Option<byte32>
```

Comment:
I think `byte32` is better and here is pseudo code, not real coding implementation

Reply:
updated